coveralls.json

```diff
@@ -3,6 +3,9 @@
       "minimum_coverage": 100
     },
     "skip_files": [
-      "test/"
+      "test/",
+      "lib/auth_mvp_web.ex",
```

Comment:
we aren't directly using these files so they aren't covered by tests hence adding them here.